index.html

```diff
@@ -0,0 +1,174 @@
+                </div>
+                <h3 class="title-3">
+                    Mobile Development </h3>
+                <p class="text-3"> We create applications for the most popular mobile platforms: iOS, Android.
+
+                    Our iOS apps have their analogues for the Android mobile platform. 
+                </p>
+
+
+            </article>
+            <article class="service-card">
+                <div class="service-img"><i class="fa fa-gamepad" aria-hidden="true"></i>
+                </div>
+                <h3 class="title-3">
+
+                    Game development </h3>
+                <p class="text-3"> Our digital marketing experts help you grow the business. Whether you want to get the
+                    first line in Google search results, </p>
+
+
+            </article>
+            <article class="service-card">
+                <div class="service-img"><i class="fa fa-users" aria-hidden="true"></i>
+                </div>
+                <h3 class="title-3">  Project Management </h3>
+                <p class="text-3"> IT Project Management is based on our realization of your ideas. We are ready to
+                   </p>
+
+
+            </article>
+           
+        </div>
+        <a href="#" class="services-link"> See all services</a>
+        </section>
+        <section class="our-team">
+          <h2 class="title-2">Our teams</h2>
+            <div class="team-conatiner" id="team-container"></div>
+
+        </section>
+        <section class="partner-conatiner">
+           <h2 class="title-2">Trusted by </h2>
+         <ul class="partners">
+             <li> <img src="assets/images/ethio-tel.jpeg" alt="partner logo" class ="partner-img"></li>
+             <li> <img src="assets/images/ethio-electric.png" alt="partner logo" class ="partner-img"></li>
+             <li><img src="assets/images/hibretbank.jpeg"  alt="partner logo" class ="partner-img"> </li>
+             <li><img src="assets/images/tebib.jpg" alt="partner logo" class ="partner-img"> </li>
+             <li><img src="assets/images/gebeya.webp" alt="partner logo"  class ="partner-img"> </li>
+             <li><img src="assets/images/tena.jpeg" alt="partner logo" class ="partner-img"> </li>
+       
+           
+         </ul>
+        </section>
+
+  </main>
+    <footer class="footer">
+        <div class="footer-logo"> 
+            <img src="./assets/images/logo.jpg" alt="Logo ">
+            <p class="company-name">Upper Tech</p>
+         </div>
+        <p>
+            A smartphone is a computer - it's not built using a computer - the job it does is the job of being a
+            computer. So, everything we say about computers, that the software you run should be free - you should
+            insist on that - applies to smart phones just the same. And likewise to those tablets.
+            <br> <br>
+            &copy; Upper Tech Allright reserved (@2022)
+        </p>
+
+
+    </footer>
```

Comment:
- [ ] The footer in the mobile version should have a white background as the Design provided so I encourage you to do that 

- [ ] Design 
![image](https://user-images.githubusercontent.com/61790587/169374567-b8a153e2-6eec-4613-a534-d63109740de9.png)

- [ ] Yours 
![image](https://user-images.githubusercontent.com/61790587/169374587-44d05d6e-f77a-46da-afec-038688a56155.png)